frame/identity/src/lib.rs

```diff
@@ -531,8 +547,12 @@ decl_module! {
 		fn clear_identity(origin) {
```

Comment:
Weight comment should be updated, cannot make a recommendation on GitHub but:

```
/// - Two storage mutations.
```

Is now 2 + n storage mutations where n is number of sub accounts.